### DISCUSSION TITLE
timestamp

### DISCUSSION THREAD

**Discussion by kengketa** [hagopj13/node-express-boilerplate#219] [General]
I there, I am struggling with return timestamps from User Schema
<img width="972" alt="Screen Shot 2565-09-11 at 21 34 21" src="https://user-images.githubusercontent.com/32511654/189533221-c5e21a8a-014c-4d1e-b39d-e31b0b7276ad.png">

is there any configuration somewhere to enable timestamps to be returned?

Ken
Appreciated your time

**Comment by alibabayev0** [comment]
Hi, u can add yourCustomTimeStam: new Date(0) to schema, or u can see more details about updatedAt and createdAt ref: https://mongoosejs.com/docs/timestamps.html

**Reply by kengketa** [reply]
thanks, I though I would be able to use the default timestamps from Mongo ><
